src/pages/event-schema.ts

```diff
@@ -15,8 +15,20 @@ export const eventDateSchema = Joi.date()
     "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
   });
 
-export const eventSchema = Joi.object({
-  eventDate: eventDateSchema,
+export const endDateSchema = Joi.date()
+  .iso()
+  .allow("")
+  .min(Joi.ref("startDate"))
+  .max(maxDate)
+  .messages({
+    "date.format": `Date must be a real date`,
+    "date.min": `The end date must be the same as or after the start date`,
+    "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
+  });
+
+export const stageSchema = Joi.object({
```

Comment:
I think we need both eventSchema and stageSchema. eventSchema is used for validation in  validate-publish-LDS-page and status-change-event-validation and stageSchema for validating the stages

Reply:
if we have both can we use eventSchema in stageSchema?
i.e.
```

const stageSchema: {
startEvent: eventSchema,
endEvent: eventSchema
}

```

Reply:
I'm not too sure I understand (also the schema for start and end date are different now)

Reply:
I was wondering whether, having an eventSchema, we could re-use the evntSchema for the stageSchema since a stage is composed of 2 events and the validation is similar

Reply:
I think the start and end dates have different validation means this doesn't work?